src/components/Articles/index.tsx

```diff
@@ -0,0 +1,72 @@
+'use client'
+import './scss/mystile.scss';
+import image from "./img/Rectangle.jpg";
+import Image from "next/image";
+import React, { useEffect, useState } from 'react';
+import { fetchData } from '../../app/async/api';
+import { BsFiletypePdf } from 'react-icons/bs';
+import { SlPaperClip } from 'react-icons/sl';
+import { FaFacebook, FaTelegram, FaWhatsapp } from 'react-icons/fa6';
+
+const ArticlesTuComponente: React.FC = () => {
+  const [articles, setAerticles] = useState<Article[]>([]);
+
+  useEffect(() => {
+    const fetchDataAndSetState = async () => {
+      const data = await fetchData();
+    //   console.log(data);
+      setAerticles(data);
+    };
+
+    fetchDataAndSetState();
+  }, []);
+
+    return (
+        <div className="flex min-h-screen items-top justify-between p-24">
+            <div className=" w-3/4 mr-16">
+                {articles.slice(0, 10).map((article: Article) => (
+                    <div key={articles.id} className="">
```

Comment:
Ok, create another component for every item of the component in a separate file.